### PR TITLE
landcover classification using unet bug resolved

### PR DESCRIPTION
**Pull request by guneetmutreja**
Contains bug fixes mentioned in issue: https://github.com/ArcGIS/geosaurus/issues/5086

### CODE REVIEW

File: samples/04_gis_analysts_data_scientists/land_cover_classification_using_unet.ipynb [hunk 1]
```diff
@@ -78,7 +78,7 @@
    "metadata": {},
```
Comment (priyankatuteja) [inline samples/04_gis_analysts_data_scientists/land_cover_classification_using_unet.ipynb:78]: <p>agol_gis or agol (lowercase variable name)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/848/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='33'/>

Comment (priyankatuteja) [inline samples/04_gis_analysts_data_scientists/land_cover_classification_using_unet.ipynb:78]: <p>agol_gis or agol (lowercase variable name)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/848/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='33'/>

**Review by priyankatuteja — CHANGES_REQUESTED** [review]
Could you resolve merge conflicts and change the variable  name to lowercase

File: samples/04_gis_analysts_data_scientists/land_cover_classification_using_unet.ipynb (hunk 1 above)
Comment (priyankatuteja) [inline samples/04_gis_analysts_data_scientists/land_cover_classification_using_unet.ipynb:78]: <p>agol_gis or agol (lowercase variable name)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/848/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='33'/>

**Comment by AtmaMani** [comment]
@guneetmutreja can you address the comments and resolve the merge conflicts?

**Comment by guneetmutreja** [comment]
@priyankatuteja  the changes you suggested, are implemented. Please review those. Thanks.

**Comment by guneetmutreja** [comment]
@priyankatuteja  the changes you suggested, are implemented. Please review those. Thanks.

**Comment by guneetmutreja** [comment]
Changes already merged with master.

**Pull request closed without merging**